hoomd/md/force.py

```diff
@@ -521,3 +521,83 @@ def _set_cpp_obj(self):
             _md, base_class_str)(sim.state._cpp_sys_def,
                                  sim.state._get_group(self.filter),
                                  self.manifold_constraint._cpp_obj)
+
+
+class Constant(Force):
+    r"""Constant force.
+
+    Args:
+        filter (`hoomd.filter`): Subset of particles on which to
+            apply constant forces.
+
+    `Constant` applies a type dependent constant force and torque on all
+    particles selected by the filter. `Constant` sets the force and torque
+    to  ``(0,0,0)`` for particles not selected by the filter.
+
+    Examples::
+
+        constant = hoomd.md.force.Constant(
+            filter=hoomd.filter.All()
+            )
+        constant.constant_force['A'] = (1,0,0)
+        constant.constant_torque['A'] = (0,0,0)
+
+    Note:
+
+        The energy and virial associated with the constant force are 0.
+
+    Attributes:
+        filter (`hoomd.filter`): Subset of particles on which to
+            apply constant forces.
+
+    .. py:attribute:: constant_force
+
+        Constant force vector in the global reference frame of the system
+        :math:`[\mathrm{force}]`.  It is defined per particle type and stays
+        constant during the simulation.
```

Comment:
```suggestion
        :math:`[\mathrm{force}]`.  It is defined per particle type and 
        defaults to (0.0, 0.0, 0.0) for all types.
```
It is not necessarily constant, the user can change it. Also, document the default value.